setup.py

```diff
@@ -39,7 +39,7 @@
         "wheel>=0.30.0,<1.0.0",
         "pandas>=1.0.0,<2.0.0",
         "pyarrow>=4.0.0,<7.0.0",
-        "click>=6.6,<8.0",
+        "click>=6.6,<9.0",
```

Comment:
can we just remove the upper bound?

Reply:
we could. I guess the idea for upper-bounds is to catch the case where the API changes between major versions, but I'm not sure if we need this kind of protection for `click`.